README.md

```diff
@@ -27,13 +27,18 @@ To install the `cocli` command, do:
 ```
 $ go install github.com/veraison/cocli@latest
 ```
-
-To configure auto-completion, use the `completion` subcommand.  For example, if
-`bash` is your shell, you would do something like:
+To configure auto-completion, use the `completion` subcommand. For example, if `bash` is your shell, you would do something like:
 ```
 $ cocli completion bash > ~/.bash_completion.d/cocli
 $ . ~/.bash_completion
 ```
+
+If you are setting up in Windows Bash, you can use:
```

Comment:
I’m unfamiliar with windows bash. Is this more common than powershell, the cmd terminal, or cygwin? Or do you mean through WSL? Please clarify when you say “windows bash”.

grammar thing: colons should not follow verbs, so please remove from “use:”.

Reply:
I’ve fixed the grammar issue you pointed out regarding the colon after verbs thank you for catching that and, I was referring to Git Bash.